assets/game_projects/fighter/scenes/main.json

```diff
@@ -89,169 +89,365 @@
-                                            "width": 30,
-                                            "height": 35
+                                            "x": 171,
+                                            "y": 20,
+                                            "width": 19,
+                                            "height": 20
+                                        }
+                                    },
+                                    {
+                                        "frame": 10,
+                                        "texture_path": "assets/game_projects/fighter/assets/fighters/puncher/puncher_basic_sheet.png",
+                                        "draw_source": {
+                                            "x": 190,
+                                            "y": 20,
+                                            "width": 19,
+                                            "height": 20
+                                        }
+                                    },
+                                    {
+                                        "frame": 11,
+                                        "texture_path": "assets/game_projects/fighter/assets/fighters/puncher/puncher_basic_sheet.png",
+                                        "draw_source": {
+                                            "x": 209,
+                                            "y": 20,
+                                            "width": 19,
+                                            "height": 20
+                                        }
+                                    },
+                                    {
+                                        "frame": 12,
+                                        "texture_path": "assets/game_projects/fighter/assets/fighters/puncher/puncher_basic_sheet.png",
+                                        "draw_source": {
+                                            "x": 228,
+                                            "y": 20,
+                                            "width": 19,
+                                            "height": 20
+                                        }
+                                    },
+                                    {
+                                        "frame": 13,
+                                        "texture_path": "assets/game_projects/fighter/assets/fighters/puncher/puncher_basic_sheet.png",
+                                        "draw_source": {
+                                            "x": 247,
+                                            "y": 20,
+                                            "width": 19,
+                                            "height": 20
+                                        }
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "WalkBack",
```

Comment:
Can you make animation names lower case and have words separated by underscores?